<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>8514oem</family>
    <pointsize>60</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 170, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>410</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8514oem</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 0);</string>
    </property>
    <property name="text">
     <string>PLAY:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drawButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>410</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8514oem</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>DRAW</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8514oem</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>MOTHER:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8514oem</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>A TEXT BASED CARD GAME</string>
    </property>
   </widget>
   <widget class="QLabel" name="player1score">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>515</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8514oem</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PLAYER 1 SCORE:</string>
    </property>
   </widget>
   <widget class="QLabel" name="player2score">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>510</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8514oem</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PLAYER 2 SCORE:</string>
    </property>
   </widget>
   <widget class="QLabel" name="player3score">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>510</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8514oem</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PLAYER 3 SCORE:</string>
    </property>
   </widget>
   <widget class="QLabel" name="rank">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8514oem</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RANK:</string>
    </property>
   </widget>
   <widget class="QLabel" name="suit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8514oem</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SUIT:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8514oem</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>QUIT  X</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="player1scoreEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>510</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 253, 197);
font: 12pt &quot;8514oem&quot;;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="player2scoreEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>510</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 253, 197);
font: 12pt &quot;8514oem&quot;;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="player3scoreEdit">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>510</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 253, 197);
font: 12pt &quot;8514oem&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8514oem</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 85, 0);</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="rankEdit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>190</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 253, 197);
font: 12pt &quot;8514oem&quot;;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="suitEdit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>250</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 253, 197);
font: 12pt &quot;8514oem&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>140</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8514oem</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>YOUR CARDS:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="yourCardsEdit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>190</y>
      <width>221</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 253, 197);
font: 12pt &quot;8514oem&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="playEdit">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>410</y>
      <width>113</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 253, 197);
font: 12pt &quot;8514oem&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8514oem</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FACE UP CARD:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>310</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8514oem</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PLAYER:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="playerEdit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>350</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 253, 197);
font: 12pt &quot;8514oem&quot;;</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>221</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8514oem</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 253, 197);</string>
    </property>
    <property name="plainText">
     <string>CARD VALUES:	
K 	40
Q	30
J 	20
2-10 	FACE VALUE
A 	11</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>221</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8514oem</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 253, 197);</string>
    </property>
    <property name="plainText">
     <string>CARD POWERS:	
K, 8 	SKIP
Q	REVERSE
J 	WILD/MULTIPLY
7	DRAW/DEFEND</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>8514oem</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 253, 197);</string>
    </property>
    <property name="plainText">
     <string>500 -&gt; 	200
250 -&gt; 	50
&gt;500 = 	OUT</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
